certbot/docs/using.rst

```diff
@@ -630,138 +630,77 @@ Follow these steps to safely delete a certificate:
-RSA public key. If a certificate is successfully renewed using
-specified options, those options will be saved and used for future
-renewals of that certificate.
-
-An alternative form that provides for more fine-grained control over the
-renewal process (while renewing specified certificates one at a time),
-is ``certbot certonly`` with the complete set of subject domains of
-a specific certificate specified via `-d` flags. You may also want to
-include the ``-n`` or ``--noninteractive`` flag to prevent blocking on
-user input (which is useful when running the command from cron).
-
-``certbot certonly -n -d example.com -d www.example.com``
-
-All of the domains covered by the certificate must be specified in
-this case in order to renew and replace the old certificate rather
-than obtaining a new one; don't forget any `www.` domains! Specifying
-a subset of the domains creates a new, separate certificate containing
-only those domains, rather than replacing the original certificate.
-When run with a set of domains corresponding to an existing certificate,
-the ``certonly`` command attempts to renew that specific certificate.
+  SLEEPTIME=$(awk 'BEGIN{srand(); print int(rand()*(3600+1))}'); echo "0 0,12 * * * root sleep $SLEEPTIME && certbot renew -q" | sudo tee -a /etc/crontab > /dev/null
 
-Please note that the CA will send notification emails to the address
-you provide if you do not renew certificates that are about to expire.
+If you needed to stop your webserver to run Certbot, you'll want to
+add ``pre`` and ``post`` hooks to stop and start your webserver automatically.
+For example, if your webserver is HAProxy, run the following commands to create the hook files
+in the appropriate directory:
 
-Certbot is working hard to improve the renewal process, and we
-apologize for any inconvenience you encounter in integrating these
-commands into your individual environment.
+.. code-block:: shell
 
-.. note:: ``certbot renew`` exit status will only be 1 if a renewal attempt failed.
-  This means ``certbot renew`` exit status will be 0 if no certificate needs to be updated.
-  If you write a custom script and expect to run a command only after a certificate was actually renewed
-  you will need to use the ``--deploy-hook`` since the exit status will be 0 both on successful renewal
-  and when renewal is not necessary.
+  sudo sh -c 'printf "#!/bin/sh\nservice haproxy stop\n" > /etc/letsencrypt/renewal-hooks/pre/haproxy.sh'
+  sudo sh -c 'printf "#!/bin/sh\nservice haproxy start\n" > /etc/letsencrypt/renewal-hooks/post/haproxy.sh'
+  sudo chmod 755 /etc/letsencrypt/renewal-hooks/pre/haproxy.sh
+  sudo chmod 755 /etc/letsencrypt/renewal-hooks/post/haproxy.sh
+
+Congratulations, Certbot will now automatically renew your certificates in the background.
+
+If you are interested in learning more about how Certbot renews your certificates, see the
+`Renewing certificates`_ section above.
```

Comment:
Given the reshuffling that has occurred here, I think this should instead link to the reference section for `certbot renew`.